input_dictionary.json

```diff
@@ -938,7 +938,83 @@
           "tooltip": "Number of decimal digits to display"
         }
       ]
+    },
+    {
+      "estimator_id": "UOVEstimator",
+      "algorithm_id": "UOVAlgorithm",
+      "display_label": "UOV Signature Scheme",
+      "landing_page_content": "# UOV Estimator\n\n\nThis project provides an estimator for the security of the Unblanced Oil and Vinegar (UOV) signature scheme. This scheme is mainly based on the hardness of the MQ problem and it was submitted as a candidate to the NIST call for post-quantum signatures. \n\nMore details on the theoretical foundations of the signature scheme and the estimator can be found in the submitted specifications:\n\n[[UOV23]](https://csrc.nist.gov/csrc/media/Projects/pqc-dig-sig/documents/round-1/spec-files/UOV-spec-web.pdf) Ward Beullens, Ming-Shing Chen, Jintai Ding, Boru Gong, Matthias J. Kannwischer, Jacques Patarin, Bo-Yuan Peng, Dieter Schmidt, Cheng-Jhih Shih, Chengdong Tao and Bo-Yin Yang. UOV: Unbalanced Oil and Vinegar\nAlgorithm Specifications and Supporting Documentation. ",
```

Comment:
I think this should also include a citation to the original UOV publication(s). Or was there a particular reason to exclude those?

Reply:
Make sense.